filebeat/harvester/config.go

```diff
@@ -6,42 +6,42 @@ import (
 	"time"
 
 	cfg "github.com/elastic/beats/filebeat/config"
-	"github.com/elastic/beats/filebeat/input"
+	"github.com/elastic/beats/filebeat/harvester/processor"
 	"github.com/elastic/beats/libbeat/common"
 )
 
 var (
 	defaultConfig = harvesterConfig{
-		BufferSize:      cfg.DefaultHarvesterBufferSize,
-		DocumentType:    cfg.DefaultDocumentType,
+		BufferSize:      16 << 10, // 16384
```

Comment:
While I can read it, I like `16 * (1 << 10)`. Or give `1<<10` a name like, `sz1K = (1<<10)`